### ISSUE TITLE
Loguru options?

### ISSUE THREAD

**Issue by arrowgent** [dosbox-staging/dosbox-staging#1470]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### Different version than latest?

0.78.0 release

### What Operating System are you using?

Linux x86_64

### Whats your question and how can we help?

https://github.com/dosbox-staging/dosbox-staging/pull/1225
loguru looks great in terminal
great choice
which upstream source are you using? is it from dosbox or fork?

are there any -switch options or conf to modify its output?
id like to reformat the time output or remove it entirely

colors working great,  shows grey time, white info
red sigfaults (ctrl+c)
ex: `Loguru caught a signal: SIGINT`
nice results.

this also appears to give far more info in the log than previously, highly useful without needing a debug build.

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by kcgen** [comment]
That's right @arrowgent - user-level verbosity control of Loguru's message isn't in place yet.

I'm tagging w/ "help wanted", as this would be nice to have someone in the community help with this. First step would be adding add verbosity control to Staging's command line, second step would be the conf system.

**Comment by kcgen** [comment]
> which upstream source are you using? is it from dosbox or fork?

Yes, I've pushed fixes for compiler errors and static analysis issues upstream: <https://github.com/emilk/loguru/pulls>, but it seems the repo is unfortunately no longer actively maintained.   Its own release branch doesn't even pass it's own CI tests because the CI system has continued to progress out from under the codebase.

So for now, I will kep pushing changes upstream as PRs in hope of the original author to return and process them.  Until then, we are keeping them here too.

You can see the network graph and drag it to the right:
 
https://github.com/emilk/loguru/network

**Comment by arrowgent** [comment]
ah ok its the C++ loguru not the more active python loguru.
yes hopefully they continue developing!

good to know loguru is working and currently initial stages within dosbox-staging
